assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
The use case is simple here. `focusId` contains ID of the element to focus. `href` might contain an id to focus or an URL to go to that's why so many checks around it. One is to see if the `href` starts with a hash. It's simple case if it doesn't - let the browser handle the click. Few lines below there's another check to see if the matching DOM element exists (break if doesn't). What issues are you referring to?